src/uct/cuda/cuda_ipc/cuda_ipc_iface.c

```diff
@@ -473,7 +473,8 @@ static uct_iface_internal_ops_t uct_cuda_ipc_iface_internal_ops = {
     .ep_query              = (uct_ep_query_func_t)ucs_empty_function_return_unsupported,
     .ep_invalidate         = (uct_ep_invalidate_func_t)ucs_empty_function_return_unsupported,
     .ep_connect_to_ep_v2   = ucs_empty_function_return_unsupported,
-    .iface_is_reachable_v2 = uct_base_iface_is_reachable_v2
+    .iface_is_reachable_v2 = uct_base_iface_is_reachable_v2,
+    .ep_is_connected       = (uct_ep_is_connected_func_t)ucs_empty_function_return_unsupported
```

Comment:
ucs_empty_function_return_unsupported returns nonzero status, so it means always connected?